pypeit/wavetilts.py

```diff
@@ -334,7 +480,36 @@ def run(self, maskslits=None, doqa=True, debug=False, show=False):
             maskslits = np.zeros(self.nslits, dtype=bool)
 
         # Extract the arc spectra for all slits
-        self.arccen, self.arc_maskslit = self.extract_arcs()#self.slitcen, self.slitmask, self.inmask)
+        self.arccen, self.arccen_bpm, self.arc_maskslit = self.extract_arcs()
+
+        # TODO: Leave for now.  Used for debugging
+#        self.par['rm_continuum'] = True
+#        debug = True
+#        show = True
+
+        # Subtract arc continuum
+        _msarc = self.msarc.image.copy()
```

Comment:
We should discuss which instruments have this rm_continuum set to true.

Reply:
At the moment, it's only VLT XShooter NIR.